File: PB - More Exercises Python/1.0 Simple Operations and Calculations - More Exercises/06. Fishland.py
skumriq_price, caca_price, kgPalamud, kgSafrid, kgMidi  = [float(input()) for _ in range(5)]
palamud_price = skumriq_price + 0.6 * skumriq_price
safrid_price = caca_price + 0.8 * caca_price
price = (kgPalamud * palamud_price + kgSafrid * safrid_price + kgMidi * 7.5)
print(f'{price:.2f}')


'''
6.90
4.20
1.5
2.5
1
'''

File: PB - More Exercises GO/1.0 Simple Operations and Calculations - More Exercises/06. Fishland.go
package main

import "fmt"

func main() {
	var skumriqPrice, cacaPrice, kgPalamud, kgSafrid, kgMidi float64
	fmt.Scan(&skumriqPrice, &cacaPrice, &kgPalamud, &kgSafrid, &kgMidi)

	palamudPrice := skumriqPrice + 0.6*skumriqPrice
	safridPrice := cacaPrice + 0.8*cacaPrice
	price := (kgPalamud*palamudPrice + kgSafrid*safridPrice + kgMidi*7.5)
	fmt.Printf("%.2f", price)
}

/*
name   :06. Fishland
input  :6.90 4.20 1.5 2.5 1
output :42.96
*/
